posthog/api/property_definition.py

```diff
@@ -33,8 +60,9 @@ class PropertyDefinitionViewSet(
     permission_classes = [permissions.IsAuthenticated, OrganizationMemberPermissions]
     lookup_field = "id"
     ordering = "name"
-    filter_backends = [filters.SearchFilter]
-    search_fields = ["name"]
+    pg_trgm_installed = is_pg_trgm_installed()
```

Comment:
This is evaluated when class is initialized, meaning whenever someone imports _anything_ we do the above db queries. That's really really bad and will make _everything_ in posthog slower and more dangerous to initialize.

We should do these instead when get_queryset or something like that.